src/jit/emitxarch.cpp

```diff
@@ -367,6 +367,8 @@ bool emitter::IsDstDstSrcAVXInstruction(instruction ins)
         case INS_vperm2f128:
         case INS_vpermilpsvar:
         case INS_vpermilpdvar:
+        case INS_vpermpd:
+        case INS_vpermq:
```

Comment:
To solve https://github.com/dotnet/coreclr/issues/19625, you need to remove these two lines.